### ISSUE TITLE
Improve support for non-standard location libraries

### ISSUE THREAD

**Issue by thommey** [eggheads/eggdrop#426]
We could use the AC_LIB_{,HAVE_}LINKFLAGS macros to get the linker -rpath into our binary to eliminate the need for LD_LIBRARY_PATH setting on linux/glibc. Not very portable, but the common case.

https://www.gnu.org/software/gnulib/manual/html_node/Searching-for-Libraries.html

Custom paths would need to be added to LDFLAGS/CPPFLAGS before calling this.

**Comment by ghost** [comment]
Would that address the issue I presented to you about [info library] pointing to default locations instead of actual library install location, or is this something completely different?

**Comment by thommey** [comment]
I don't believe it will, I'll add a comment in #345 about the state of that.
